### PR TITLE
Infobox text and URI change

### PR DESCRIPTION
**Pull request by q4wen**
In search info box, the text and link of HGNC are changed (ref. #756 ):

1. Text: "HGNC Symbol"
2. URI format: `http://identifiers.org/hgnc.symbol/<HGNC approved symbol>`

Before:
link-before
<br/>
<br/>
After:
screen shot 2018-05-15 at 3 09 31 pm

### CODE REVIEW

**Comment by maxkfranz** [comment]
Great!  I think it makes sense for the text in the UI to just say "HGNC".  Users don't really care how the link is made.

**Comment by q4wen** [comment]
@d2fong The text and link are updated. Could you review?

File: src/client/features/search/landing-box.js [hunk 1]
```diff
@@ -47,9 +47,18 @@ const pcFallback = (unrecognized,genes) => {
...
-  return _.assign({},
+  let dbs =  _.assign({},
     ...dbSet.map(database=>({[database.database]:database.url+database.search+ids[database.database].replace(/[^a-zA-Z0-9:]/g)}))
   );
+
+  let links = [];
+  usedDatabases.forEach((usedDatabase)=>{
+    let dbLink = dbs[usedDatabase.configName];
+    if (dbLink != null) {
+      links.push({"link":dbLink,"displayName":usedDatabase.displayName});
```
Comment (d2fong) [inline src/client/features/search/landing-box.js]: 1. This is a minor nitpick but to keep the coding style consistent: for object keys you generally don't need to use strings you can just do something like this.
``` js
links.push({ link: dbLink, displayName: usedDatabase.displayName });
```

File: src/client/features/search/landing-box.js [hunk 2]
```diff
@@ -8,10 +8,10 @@ const Loader = require('react-loader');
 const _ = require('lodash');
 //usedDatabases=[['Uniprot lookup name',{configName:,gProfiler:}]]
 const usedDatabases=new Map ([
-  ['GeneCards',{configName:'Gene Cards',gProfiler:'HGNCSYMBOL'}],
-  ['HGNC',{configName:'HGNC',gProfiler:'HGNC'}],
-  ['GeneID',{configName:'NCBI Gene',gProfiler:'NCBIGene'}],
-  ['Uniprot',{configName:'Uniprot',gProfiler:'Uniprot'}]
+  ['GeneCards',{configName:'Gene Cards',gProfiler:'HGNCSYMBOL',displayName:'Gene Cards'}],
+  ['HGNC Symbol',{configName:'HGNC Symbol',gProfiler:'HGNCSymbol',displayName:'NGNC'}],
```
Comment (jvwong) [inline src/client/features/search/landing-box.js]: Typo in 'NGNC' --> 'HGNC'?

File: src/client/features/search/landing-box.js (hunk 1 above)
Comment (maxkfranz) [reply src/client/features/search/landing-box.js]: The only place where quotation marks are mandatory on keys is in JSON files.

**Pull request merged**